services/kube-prometheus-stack/48.3.1/defaults/cm.yaml

```diff
@@ -334,15 +334,14 @@ data:
             memory: 200Mi
     grafana:
       enabled: true
-      # pinning grafana to 9.5 since it appears that upgrading to v10 is causing issues with prometheus scraping
-      # grafana metrics
       image:
-        tag: 9.5.13
+        tag: 10.3.3
```

Comment:
can we just remove this config and let the default (with this version of KPS) be used?

Reply:
done. also verified metrics scraping works in the default version (10.0.3)